### DISCUSSION TITLE
proxy support for svelte

### DISCUSSION THREAD

**Discussion by xsintill** [nxext/nx-extensions#305] [Ideas]
Dominik would it be possible to add proxy support to the svelte package? The serve rollup plugin currently used has no proxy support. This would help communication with a locally developed nestjs server (also in nx). More people would probably benefit from this 

The plugin I currently use is:
rollup-plugin-serve-proxy
in rollup-options.js replacing the line 
`const serve = require('rollup-plugin-serve');`
with
`const serve = require('rollup-plugin-serve-proxy');`

adding this line to the serveOptions object in the rollup-options.js
`proxy: options.proxy`

in the workspace.json
```
"your-svelte-client": {
   ...
    "serve": {
       ...
       "option": {
          ...
         "proxy": {
              "api": "http://localhost:3000"
          },
       }
    }
}
```

**Comment by Jordan-Hall** [comment]
@DominikPieper can get a vite board for these things and have a direct replacement for webpack

**Reply by DominikPieper** [reply]
@Jordan-Hall the feature itself is there for the rollup soluton. We should address this as well for the Vite replacement

**Comment by Jordan-Hall** [comment]
This is now fully implemented thanks to using VITE https://nxext.dev/docs/vite/executors/serve#--proxyconfig I'm going to go ahead and lock this. 

Thank you @xsintill  for the suggestion
